source/features/latest-tag-button.tsx

```diff
@@ -56,34 +65,27 @@ const getRepoPublishState = cache.function(async (): Promise<RepoPublishState> =
 		tags.set(node.name, node.tag.commit?.oid ?? node.tag.oid);
 	}
 
+	const repoCommits = new Map<string, string>();
+	for (const [index, {oid}] of repository.defaultBranchRef.target.history.nodes.entries()) {
+		repoCommits.set(oid, index);
+	}
+
 	const latestTag = getLatestVersionTag([...tags.keys()]);
+	const latestTagOid = tags.get(latestTag)!;
 
 	return {
 		latestTag,
-		isUpToDate: tags.get(latestTag) === repository.defaultBranchRef.target.oid
+		isUpToDate: tags.get(latestTag) === repository.defaultBranchRef.target.oid,
+		aheadBy: repoCommits.get(latestTagOid)! ?? '20+'
 	};
 }, {
 	maxAge: 1 / 24, // One hour
 	staleWhileRevalidate: 2,
-	shouldRevalidate: value => typeof value === 'string',
```

Comment:
This PR is changing the format again, so this should have another test. I needs a `TODO: remove after June` too

Reply:
>so this should have another test.

I dont understand what you mean

Reply:
What about 6cd1c86698ba6f525b20258384c60c0d7f697437

Reply:
```js
	shouldRevalidate: value => 'isUpToDate' in value, // TODO: remove after June
```

Reply:
Should I change the cache key too? Since I know you dont like when its __filebasename